v1/package.json

```
@@ -13,7 +13,7 @@
     "dev": "ts-node-esm index.ts | pino-pretty -c -t",
     "watch": "nodemon --watch './**/*.ts' --exec 'node --experimental-specifier-resolution=node --loader ts-node/esm' index.ts | pino-pretty -c -t",
     "start": "node build/index.js",
-    "test": "ava --timeout=5m"
+    "test": "ava --timeout=2m"
```

```suggestion
    "test": "ava --timeout=5m"
```

Sadly IPNS updates take forever so I think we should keep the timeout at 5 minutes to be safe. Maybe 4 minutes (3 mins means we'll get false negatives sometimes)